src/main/kotlin/com/pubnub/api/models/server/history/ServerFetchMessageItem.kt

```diff
@@ -0,0 +1,18 @@
+package com.pubnub.api.models.server.history
+
+import com.google.gson.JsonElement
+import com.google.gson.annotations.SerializedName
+import com.pubnub.api.models.consumer.history.Action
+
+data class ServerFetchMessageItem(
+    val uuid: String?,
+    val message: JsonElement,
+    val meta: JsonElement?,
+    val timetoken: Long,
+    val actions: Map<String, Map<String, List<Action>>>? = null,
+    @SerializedName("message_type")
+    val integerMessageType: Int? = null,
```

Comment:
What about changing it to pnMessageType  or pnMessageTypeInt for consistence with other usages e.g. in SubscribeMessage there is pnMessageTypeInt. ?

Reply:
Yes. I missed this one :(